src/plugins/colum-chooser/column-chooser.module.ts

```diff
@@ -0,0 +1,32 @@
+import {NgModule} from '@angular/core';
+import {BrowserModule} from '@angular/platform-browser';
+import {TemplateModule} from 'ng2-qgrid/template';
+import {FormsModule} from '@angular/forms';
+import {ColumnChooserComponent} from './column-chooser.component';
+import { Model } from 'ng2-qgrid/core/infrastructure/model';
+
+import {ColumnChooserModel} from './column-chooser.model';
+
+import {
```

Comment:
all templates should be in theme/material

Reply:
fixed